lib/statistics.rb

```diff
@@ -2,23 +2,35 @@
 
 # This module contains a number of methods to calculate various statistics
 module Statistics
-  def self.mom_changed_cases_growth(cases_at_start:, cases_at_end:)
-    return 0 if cases_at_end.size.zero?
+  def self.mom_changed_metric_growth(metric_at_start:, metric_at_end:)
+    return 0 if metric_at_end.size.zero?
 
-    case_growth = cases_at_start.size - cases_at_end.size
-    return (cases_at_end.size * 100) if case_growth.zero?
+    metric_growth = metric_at_start.size - metric_at_end.size

```

Comment:
It looks like lines 8 - 9, 17 & 18 and 30, 31 can be refactored into another method called 
`def self.metric_growth(metric_start, metric_end)
  growth = metric_start.size - metric_end.size
  (growth.zero?) ? metric_end.size * 100 : growth 
end
```

I think it can help abstract that out to a separate function to see how the others exist with the refactored form.